release/models/platform/openconfig-platform.yang

```diff
@@ -542,6 +548,17 @@ module openconfig-platform {
         device, one as primary and the other as secondary, should both
         report the same value.";
     }
+
+    leaf base-mac-address {
+      type oc-yang:mac-address;
+      description
+        "This is a MAC address representing the root or primary MAC
+        address for a component.  Only components such as CHASSIS and
+        CONTROLLER_CARD are expected to providea base-mac-address.  The
+        base mac-address for CHASSIS and a PRIMARY CONTROLLER_CARD may
```

Comment:
Is there a reason that only the primary controller card and not the secondary can have the same `base-mac-address`? Does this imply that when there is a switchover of controller cards there would be a change in base MAC address? (Seems a challenging idea!)

Reply:
Good question.  I am collecting some more information on the use case so I can clarify appropriately.

Reply:
Updated description with use case.